src/packages/client/src/__tests__/integration/happy/blog-env-mssql/test.ts

```diff
@@ -51,10 +51,6 @@ describe('blog-env-mssql', () => {
     expect(prismaVersion.client).not.toBeUndefined()
   })
 
-  test('does not leak connection strings in node_modules', () => {
-    expect(prisma.internalDatasources).toBeUndefined()
```

Comment:
Why removing this? Is this done in another test now or not relevant anymore?

Reply:
We don't have them anymore.